Widgets/Widgets/TopReadWidget.swift

```diff
@@ -89,6 +89,14 @@ final class TopReadData {
                 group.leave()
                 continue
             }
+            
+            if usingImageCache {
+                if let cachedImage = fetcher.memoryCachedImage(withURL: thumbnailURL) {
+                    rankedElements[index].image = cachedImage.staticImage
+                    group.leave()
+                    continue
+                }
```

Comment:
It looks like `fetcher.fetchImage()` checks the memory cache first so the behavior would be the same if this goes on to fetch the image from the network. Should the continue be outside of the `if let cachedImage...` block so it doesn't go on to fetch the image from the network?

Reply:
@joewalsh well I was thinking if there was no memory cached image, then fallback to the network just as a backup to try to get the image, but I guess that defeats the purpose of a quick method for `getSnapshot`. I didn't catch that `fetchImage` leans on memory anyways. I could move the continue, but keep this commit as a whole, so we are sure a fallback/network hit never occurs? Or if we are fine hitting the network as a backup, I could revert this commit & remove `usingImageCache` entirely.

Reply:
Looking at it again, it seems the intent of the snapshot is to be as quick as possible, so it makes sense to not hit the network for images. I pushed an update in f8210e6...40d5bae that only hits the memory and file cache for snapshots, for the content itself as well as the images.